src/lib/account-profile/account-profile-menu/account-profile-menu.component.ts

```diff
@@ -0,0 +1,17 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'vui-account-profile-menu',
+  templateUrl: './account-profile-menu.component.html',
+  styleUrls: ['./account-profile-menu.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class VantageAccountProfileMenuComponent {
+  @Input() email: string;
+  @Input() name: string;
+
+  _blockEvent(event: Event): void {
```

Comment:
private?

Reply:
is used in the template to block the menu from hiding like in app switcher. cant be private if used in template

Reply:
Oops.